src/test/s3/util/fileViewerManager.test.ts

```diff
@@ -144,6 +144,12 @@ describe('FileViewerManager', function () {
     let commands: typeof vscode.commands
     let settings: SettingsConfiguration
 
+    async function closeAllEditors() {
+        // Derived by inspecting 'Keyboard Shortcuts' via command `>Preferences: Open Keyboard Shortcuts`
+        // Tests will pass without it but good to clean state regardless
+        await vscode.commands.executeCommand('openEditors.closeAll')
```

Comment:
what about the one in `src/shared/utilities/vsCodeUtils.ts` :)

Reply:
Looks like that one will need to be a fallback case.